merino-cache/src/redis/mod.rs

```diff
@@ -125,6 +130,99 @@ where
         }
     }
 
+    /// Generate a lock identifier key.
+    /// This is a VERY simple locking mechanism. The only bit of fancy is that it will
+    /// expire, allowing for "stuck" queries to eventually resolve.
+    fn pending_key(key: &str) -> String {
+        format!("pending_{}", key)
+    }
+
+    /// See iff a record update is pending. (Does not check lock value, only if it exists)
```

Comment:
Is this `iff` a typo? I don't think "if and only if" applies here.

Reply:
yep, typo. Thanks!